index.html

```diff
@@ -3,16 +3,85 @@
 <head>
   <meta charset="UTF-8">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>AJAX Project</title>
+  <title>Brewery Finder - AJAX Project</title>
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.0.0/css/all.min.css"
+    integrity="sha512-9usAa10IRO0HhonpyAIVpjrylPvoDwiPUiKdWk5t3PyolY1cOd4DSE0Ga+ri4AuTroPR5aQvXU9xC6qOPnzFeg=="
+    crossorigin="anonymous" referrerpolicy="no-referrer" />
   <link rel="icon" href="images/favicon.ico">
   <link rel="stylesheet" href="css/reset.css">
   <link rel="stylesheet" href="css/layout.css">
   <link rel="stylesheet" href="css/styles.css">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Teko:wght@400&display=swap" rel="stylesheet">
 </head>
 <body>
   <header>
+    <div class="header-background">
+      <a class="return"><h1 class="header-name">BEER RUN</h1></a>
+      <p class="subheader">Thirsty? We can fix that.</p>
+    </div>
   </header>
   <main>
+    <div class="container home-page">
+      <div class="row">
+        <div class="col-full">
+          <h2 class="intro">Let us help you find your next favorite brewery.</h2>
+        </div>
+      </div>
+      <div class="row">
+        <div class="col-full">
+          <img src="./images/beerRunGlasses.png" class="home-photo" alt="glasses of beer on a counter">
+        </div>
+      </div>
+      <div class="row">
+        <div class="col-full">
+          <form class="form-city">
+            <h2 class="city-intro">Find a brewery in your city, or any city.</h2>
+            <label for="city-name">City Name</label>
+            <input type="text" name="city-name" class="textbox">
```

Comment:
The `input` needs an `id` that matches the `for` of your label. The `for` attribute is linked by `id`, not `name`